__tests__/index.js

```diff
@@ -132,4 +132,30 @@ describe('storage', () =>
       expect(storage.toString()).toEqual('[object Storage]');
       expect(storage.toString).toHaveBeenCalledTimes(1);
     });
+
+    test('storage setItem throws', () => {
```

Comment:
This test is really unnecessary, it was just to demonstrate that it does behave correctly without reset